sites/skeleton.dev/src/routes/blog/[slug]/+page.svelte

```diff
@@ -19,11 +19,12 @@
 		// Element Page
 		elemPage = document.querySelector('#page');
 		// CodeBlock Highlight
-		document.querySelectorAll('pre code').forEach((elem: any) => {
+		document.querySelectorAll('pre code').forEach((elem: Element) => {
+			if (!(elem instanceof HTMLElement)) return;
 			hljs.highlightElement(elem);
 		});
 		// Table
-		document.querySelectorAll('table').forEach((elem: any) => {
+		document.querySelectorAll('table').forEach((elem: Element) => {
```

Comment:
```suggestion
		document.querySelectorAll('table').forEach((elem) => {
```

these can just be inferred as well